view/templates/htconfig.tpl

```diff
@@ -19,16 +19,21 @@ $db_data = '{{$dbdata}}';
 
 // Use environment variables for mysql if they are set beforehand
 if (!empty(getenv('MYSQL_HOST'))
-   && !empty(getenv('MYSQL_PORT'))
-   && !empty(getenv('MYSQL_USERNAME'))
-   && !empty(getenv('MYSQL_PASSWORD'))
-   && !empty(getenv('MYSQL_DATABASE'))) {
+	&& !empty(getenv('MYSQL_PORT'))
```

Comment:
Are they necessary in the template file for the installation wizard?

Reply:
I think so. As far as I can see the environment variables overrule the local settings so in case you change your db credentials you won't have to change the .htconfig.php as well.

On the other hand it's not clear from the .htconfig.php perspective which credentials we are now using. But this is a general problem with these environment variables I think ;-)

Reply:
Does the installation wizard check for the environment vars and skips the step if they are set?

Reply:
Yes. It is the whole point of automatic install.

Reply:
nope he doesn't. He creates the db_* variables in the .htconfig.php

Reply:
@MrPetovan but not for the installation wizard. So if you doesn't execute `bin/console autoinstall` and instead use the wizard, it creates the variables (checked it at my local environment)

Reply:
I deleted my irrelevant after seeing your answer. 😅